lib/open-amp/Kconfig

```diff
@@ -18,3 +18,19 @@ config OPENAMP_RSC_TABLE_NUM_RPMSG_BUFF
 	help
 	  This option specifies the number of buffer used in a Vring for
 	  interprocessor communication
+
+config IPM_RX_CHANNEL_ID
```

Comment:
related to the resource table not to iPM so config should be renamed
`OPENAMP_RSC_TABLE_IPM_RX_ID` or something similar